jenkins/Jenkinsfile-release

```diff
@@ -190,35 +190,35 @@ pipeline {
             }
         }
 
-//        stage('Build Conda Packages'){
-//              steps {
-//                    script {
-//                        docker.withRegistry("http://harbor.h2o.ai") {
-//                            docker.image('opsh2oai/sparkling_water_tests:9').inside("--init --dns 172.16.0.200") {
-//                                dir("py/build/conda") {
-//                                    def PYTHON_VERSIONS = ['2.7', '3.6']
-//                                    for (pyVersion in PYTHON_VERSIONS) {
-//
-//                                        sh """
-//                                            . /envs/h2o_env_python3.6/bin/activate
-//
-//                                            conda build h2o_pysparkling_2.3 --output-folder "." --no-anaconda-upload --py ${pyVersion}
-//
-//                                            # Get name of the package
-//                                            CONDA_PKG_CURRENT_ARCH_PATH=\$(conda build h2o_pysparkling_2.3 --py ${pyVersion} --output-folder "." --output | tail -1)
-//                                            PKG_NAME=\$(basename \$CONDA_PKG_CURRENT_ARCH_PATH)
-//
-//
-//                                            # Convert conda package for all other platforms
-//                                            conda convert \$CONDA_PKG_CURRENT_ARCH_PATH -p all
-//                                           """
-//                                    }
-//                                }
-//                            }
-//                        }
-//                    }
-//              }
-//        }
+        stage('Build Conda Packages'){
+              steps {
+                    script {
+                        docker.withRegistry("http://harbor.h2o.ai") {
+                            docker.image('opsh2oai/sparkling_water_tests:9').inside("--init --dns 172.16.0.200") {
+                                dir("py/build/conda") {
+                                    def PYTHON_VERSIONS = ['2.7', '3.6']
+                                    for (pyVersion in PYTHON_VERSIONS) {
+
+                                        sh """
+                                            . /envs/h2o_env_python3.6/bin/activate
```

Comment:
just out of curiosity - is this okay to use conda from python `3.6` to build a python `2.7` conda packages?

Reply:
yes